homeassistant/components/samsungtv/media_player.py

```diff
@@ -71,14 +73,15 @@
     ):
         turn_on_action = hass.data[DOMAIN][ip_address][CONF_ON_ACTION]
         on_script = Script(hass, turn_on_action)
-    async_add_entities([SamsungTVDevice(config_entry, on_script)])
+    async_add_entities([SamsungTVDevice(hass, config_entry, on_script)])
 
 
 class SamsungTVDevice(MediaPlayerDevice):
     """Representation of a Samsung TV."""
 
-    def __init__(self, config_entry, on_script):
+    def __init__(self, hass, config_entry, on_script):
```

Comment:
Don't pass in `hass`. It will be set on the entity when it has been added to home assistant.

Reply:
> nt.

Reply:
Bump.

Reply:
done

Reply:
Am I the only one still seeing this?